media-fonts/font-misc-ethiopic/font-misc-ethiopic-1.0.3-r2.ebuild

```diff
@@ -0,0 +1,23 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+FONT_DIR="OTF"
+
+inherit xorg-3
+
+DESCRIPTION="Miscellaneous Ethiopic fonts"
+
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~m68k ~mips ~ppc ~ppc64 ~s390 ~sparc ~x86"
+
+FONT_OPTIONS="--with-otf-fontdir=${EPREFIX}/usr/share/fonts/OTF"
+
+DOCS=( README ChangeLog )
+
+src_install() {
+	einstalldocs
+	xorg-3_src_install
+	# TTF fonts are not supposed to be installed.
+	# Also fixes file collision per bug #309689
+	rm -r "${ED}/usr/share/fonts/TTF" || die
```

Comment:
You changed `rm -rf` to `rm -r` which seems unexpected. I'm unsure whether any of it will be needed with a new upstream release, so I'm leaving this comment so we can recheck.

Reply:
Well, I was using this ebuild as an example for removing empty variables in #gentoo-dev-build and an "@" told me to drop the "-f", so I complied :)

Reply:
Keeping it "rm -rf" for now since you are going to release a new revision anyway.